### DISCUSSION TITLE
Bandwidth exceeded while using LFS

### DISCUSSION THREAD

**Discussion by hrishikesh-k** [community/community#22173] [New to GitHub]
<p>Hello.</p>
<p>I’m developing a portfolio website with Hugo. I have created a public repository in my account and connected it to Netlify for automatic deployment.</p>
<p>I had to include a video that was a little over 100 MB in size. So, I enabled Git LFS for my repository. However, later, I found out that I could further compress the video without much quality loss and now it’s down to 56.7 MB. So, I removed Git LFS from my repository and also removed all files that were using LFS. Now, I don’t have any file in my repository that uses LFS. Just to be sure, I also removed all commit history from Github and checked that no file now uses Git LFS.</p>
<p>It is after this that I discovered that Git LFS has bandwidth limitations. When I did, I went to my Billing section and it showed me around 750 MB was used. I thought, great, now that I disabled LFS in time, I need not worry about it. But, soon after I pushed some changes and Netlify deployed my website, I got an e-mail saying 80% bandwidth used. I thought, okay, might take some time to update the LFS status.</p>
<p>Today, I was shocked to receive an e-mail that said 100% bandwidth used. I don’t understand, I don’t use LFS any more right? And now with commit history gone, it’s not even in the records? Sorry, I haven’t understood this LFS concept, but, does LFS bandwidth get counted even without using LFS (but storing files above 50 MB)?</p>
<p>It’d be nice if someone can explain how this works and suggest what I should do now. Should I delete the repository and create another, or I can’t use GitHub ⇿ Netlify integration until I’m having files beyond 50 MB?</p>
<p>EDIT: I went ahead and tried pushing changes and publishing my website and it still works. Billing does show even more bandwidth used, but, it seems to work. Would it be creating problems down the line?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi :wave:</p>
<p>To remove files from LFS tracking, we’ll need to delete them from the server on our end. Deleting them from the GitHub repository doesn’t automatically remove them from using your storage allotment.<br>
I requested to have this done for you and it has been completed.</p>

**Comment by hrishikesh-k** [comment]
<p>Great. Thanks a lot for the update!</p>

**Comment by walker-21** [comment]
<p>I have currently deleted the repository that i used git lfs with.<br>
Can you explain how to delete files from git server.</p>

**Comment by BHavikJ12** [comment]
Please tell how can we remove it from the server.

**Comment by kyzhao-ivy** [comment]
@ernest-phillips Hi, I am wondering how to delete LFS files from the server. Thanks.

I would like to remove all files from LFS under the handle `kyzhao-ivy`, thanks!!!
